### ISSUE TITLE
Suggestion - Configuration to pass `no-skipped-test` if a conditional is provided

### ISSUE THREAD

**Issue by Johoseph** [playwright-community/eslint-plugin-playwright#157]
Conditionally skipped tests are reported the same as other skipped tests (as you have documented [here](https://github.com/playwright-community/eslint-plugin-playwright/blob/main/docs/rules/no-skipped-test.md)). It would be useful to be able to configure the linter to allow this and not flag these cases. See below for our use case 🙂

```ts
test("Example Test", () => {
  test.skip(envVarConditional); // eslint flags this

  // ... continue if conditional fails
});
```
